python/doc/theory/reliability_sensitivity/form_approximation.rst

```diff
@@ -3,106 +3,73 @@
+              E(-\beta) & \mbox{if the origin fails}, \\
+              E(+\beta) & \mbox{otherwise}
             \end{array}
             \right.
 
-   | where :math:`\beta_{HL}` is the Hasofer-Lind reliability index,
-     defined as the distance of the design point
-     :math:`\vect{u}^*` to the origin of the standard space and
-     :math:`E` the marginal cumulative density function of the spherical
-     distributions in the standard space.
-   | Let us recall here that in the Rosenblatt standard
-     space, random vectors follow the standard normal distribution (with
-     zero mean, unit variance and unit correlation matrix), which
-     implies that :math:`E = \Phi`. In the Generalized Nataf standard
-     space, random vectors follow some spherical distributions, with
-     zero mean, unit variance, unit correlation matrix and which type
-     :math:`\psi` is the one of the copula of the physical random vector
-     :math:`\vect{X}` : in that case, :math:`E` is the 1D cumulative
-     distribution function with zero mean, unit variance and which type
-     is :math:`\psi`.
-
-Here, the event considered is explained directly from the limit state
-function :math:`g(\vect{X}\,,\,\vect{d})` : this is the classical
-structural reliability formulation.
-However, if the event is a threshold exceedance, it is useful to
-explicit the variable of interest
-:math:`Z=\tilde{g}(\vect{X}\,,\,\vect{d})`, evaluated from the model
-:math:`\tilde{g}(.)`. In that case, the event considered, associated to
-the threshold :math:`z_s` has the formulation:
-:math:`\cD_f = \{ \vect{X} \in \Rset^n \, / \, Z=\tilde{g}(\vect{X}\,,\,\vect{d}) > z_s \}`
-and the limit state function is :
-:math:`g(\vect{X}\,,\,\vect{d}) = z_s - Z = z_s - \tilde{g}(\vect{X}\,,\,\vect{d})`.
-:math:`P_f` is the threshold exceedance probability, defined as :
-:math:`P_f = P(Z \geq z_s) = \int_{g(\vect{X}\,,\,\vect{d}) \le 0}  \pdf\, d\vect{x}`.
+where :math:`\beta` is the Hasofer-Lind reliability index, defined as the distance of the design point
+:math:`P^*` to the origin of the standard space and :math:`E` the marginal cumulative density function along any direction of
+the spherical distribution in the standard space (refer to :ref:`nataf_transformation` and :ref:`rosenblatt_transformation`).
 
+Let us recall here that in the Rosenblatt standard space, random vectors follow the standard
+normal
+distribution (with zero mean, unit variance and unit correlation matrix), which implies that
+:math:`E = \Phi` where :math:`Phi` is the CDF of the normal distribution with zero mean and unit variance. In the Generalized Nataf standard space, random vectors follow a spherical
+distribution, with zero mean, unit variance, unit correlation matrix and whose type :math:`\psi` is the type of the copula of the physical random vector :math:`\vect{X}`: in that case, :math:`E` is the 1D cumulative  distribution function with zero mean, unit variance and with type :math:`\psi`.
```

Comment:
The "References" section below is poorly formatted:

![image](https://user-images.githubusercontent.com/31351465/235197859-0b8f3ac4-4339-4242-838f-38df339c3935.png)

Actually, most theory help pages in this PR have the same issue.

Reply:
ok